Bug when running on k8s 1.11.9

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
Run application

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
See code output below

**Versions (please complete the following information):**
 - Linux
 - Popeye [0.1.3]
 - K8s [1.11.9]

**Additional context**
Add any other context about the problem here.

```
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat(0x146b985, 0x1, 0xfffffffffffffffd, 0xc00036a76c, 0x3)
	/usr/local/Cellar/go/1.12.1/libexec/src/strings/strings.go:533 +0x5ca
github.com/derailed/popeye/internal/report.Write(0x1647f20, 0xc00061fd80, 0x2, 0x1, 0xc000288460, 0x4e)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/internal/report/writer.go:81 +0x19c
github.com/derailed/popeye/pkg.(*Popeye).printReport(0xc000769d40, 0x7fb77196e648, 0xc0000f8240, 0xc00035ad74, 0x4)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/pkg/popeye.go:130 +0x7a5
github.com/derailed/popeye/pkg.(*Popeye).Sanitize(0xc000769d40)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/pkg/popeye.go:74 +0x4e4
github.com/derailed/popeye/cmd.doIt(0x21aa800, 0xc000203540, 0x0, 0x2)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/cmd/root.go:56 +0xdf
github.com/spf13/cobra.(*Command).execute(0x21aa800, 0xc00003a0a0, 0x2, 0x2, 0x21aa800, 0xc00003a0a0)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x21aa800, 0x0, 0x0, 0xc000381f88)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/derailed/popeye/cmd.Execute()
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/cmd/root.go:43 +0x32
main.main()
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/main.go:19 +0x20
```

--- a/internal/report/writer.go
+++ b/internal/report/writer.go
@@ -78,6 +78,9 @@ func Write(w io.Writer, l linter.Level, indent int, msg string) {
 
 	if indent == 1 {
 		dots := reportWidth - len(msg) - tabSize*indent - 3
+		if dots < 0 {
+			dots = 1
+		}
 		msg = Colorize(msg, colorForLevel(l)) + Colorize(strings.Repeat(".", dots), ColorGray)
 		fmt.Fprintf(w, "%s· %s%s\n", spacer, msg, EmojiForLevel(l))
 		return

--- a/internal/report/writer_test.go
+++ b/internal/report/writer_test.go
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
+	"strings"
 	"testing"
 
 	"github.com/derailed/popeye/internal/linter"
@@ -36,6 +37,11 @@ func TestWrite(t *testing.T) {
 			"Yo mama",
 			1,
 			"  · \x1b[38;5;155;mYo mama\x1b[0m\x1b[38;5;250;m....................................................................\x1b[0m✅\n",
+		},
+		{
+			strings.Repeat("#", reportWidth),
+			1,
+			"  · \x1b[38;5;155;m" + strings.Repeat("#", reportWidth) + "\x1b[0m\x1b[38;5;250;m.\x1b[0m✅\n",
 		},
 		{
 			"Yo mama",

